tools/dissector/packet-opendds.cpp

```diff
@@ -290,8 +290,9 @@ namespace OpenDDS
       offset += len;
 
       // hf_sequence
-      size_t size = 0, padding = 0;
-      gen_find_size(header.sequence_, size, padding);
+      size_t size = 0;
+      const Encoding encoding(Encoding::KIND_CDR_UNALIGNED);
```

Comment:
Could you factor the encoding objects out into a common one?

Reply:
Depends.  There are multiple functions.  I thought I did factor out to the minimum but I can take another look.

Reply:
Or are you saying to try and make it a single variable they all have access to within the namespace?

Reply:
> Or are you saying to try and make it a single variable they all have access to within the namespace?

Yes, I think you can just define a single encoding object for these cases and put it in the anonymous namespace higher up in the file.